File: graphify/tests/fixtures/sample.java
import java.util.List;
import java.util.ArrayList;

public class DataProcessor {
    private List<String> items;

    public DataProcessor() {
        this.items = new ArrayList<>();
    }

    public void addItem(String item) {
        items.add(item);
    }

    public List<String> process() {
        return validate(items);
    }

    private List<String> validate(List<String> data) {
        List<String> result = new ArrayList<>();
        for (String s : data) {
            if (s != null && !s.isEmpty()) {
                result.add(s.trim());
            }
        }
        return result;
    }
}

interface Processor {
    List<String> process();
}

File: graphify/tests/fixtures/sample.cs
using System;
using System.Collections.Generic;
using System.Net.Http;

namespace GraphifyDemo
{
    public interface IProcessor
    {
        List<string> Process(List<string> items);
    }

    public class DataProcessor : IProcessor
    {
        private readonly HttpClient _client;

        public DataProcessor()
        {
            _client = new HttpClient();
        }

        public List<string> Process(List<string> items)
        {
            return Validate(items);
        }

        private List<string> Validate(List<string> items)
        {
            var result = new List<string>();
            foreach (var item in items)
            {
                if (!string.IsNullOrEmpty(item))
                    result.Add(item.Trim());
            }
            return result;
        }
    }
}
